### PR TITLE
Added several unimplemented Goblin cards.

### PR DESCRIPTION
**Pull request by BursegSardaukar**

### CODE REVIEW

File: Mage.Sets/src/mage/sets/lorwyn/FodderLaunch.java [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+        this.expansionSetCode = "LRW";
+        this.subtype.add("Goblin");
+        
+        //As an additional cost to cast Fodder Launch, sacrifice a Goblin.
+        this.getSpellAbility().addCost(new SacrificeTargetCost(new TargetControlledCreaturePermanent(1, 1, filter, false)));
+        
+        //Target creature gets -5/-5 until end of turn. Fodder Launch deals 5 damage to that creature's controller.
+        this.getSpellAbility().addEffect(new BoostTargetEffect(-5, -5, Duration.EndOfTurn));
+        this.getSpellAbility().addTarget(new TargetCreaturePermanent());
+        this.getSpellAbility().addEffect(new FodderLaunchEffect());
```
Comment (LoneFox78) [inline Mage.Sets/src/mage/sets/lorwyn/FodderLaunch.java]: No need for a custom effect here. Use DamageTargetControllerEffect instead.

File: Mage.Sets/src/mage/sets/lorwyn/QuillSlingerBoggart.java [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+        super(ownerId, 137, "Quill-Slinger Boggart", Rarity.COMMON, new CardType[]{CardType.CREATURE}, "{3}{B}");
+        this.expansionSetCode = "LRW";
+        this.subtype.add("Goblin");
+        this.subtype.add("Warrior");
+
+        this.power = new MageInt(3);
+        this.toughness = new MageInt(2);
+
+        // Whenever a player casts a Kithkin spell, you may have target player lose 1 life.
+        Ability ability = new SpellCastAllTriggeredAbility(new LoseLifeTargetEffect(2), filter, true);
```
Comment (LoneFox78) [inline Mage.Sets/src/mage/sets/lorwyn/QuillSlingerBoggart.java]: 1 life, not 2

File: Mage.Sets/src/mage/sets/planeshift/MoggJailer.java [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+        super(card);
+    }
+
+    @Override
+    public MoggJailer copy() {
+        return new MoggJailer(this);
+    }
+}
+
+class CantAttackIfDefenderControllsPermanent extends RestrictionEffect {
```
Comment (LoneFox78) [inline Mage.Sets/src/mage/sets/planeshift/MoggJailer.java]: There are four other (currently unimplemented) cards that need this, so maybe it is better to put it into its own file already. Also, "Controls" is misspelled in tha name of the class.

**Comment by LevelX2** [comment]
Merged and updated the cards with commit 87078057e0bf025c078818950adf17ca18c482fe.
If you have any questions concerning the changes feel free to ask here.

Thanks for your contribution.

If you like to have a Card Tracker profile let me know your email adress or forum profile.
There you can mark the cards you are going to implement to prevent multiple implementation of the same card.
http://ct-magefree.rhcloud.com/

**Comment by BursegSardaukar** [comment]
Yea, cool, I would like a tracker profile. I'll keep experimenting with the
Goblin cards for the time being as I'm getting used to the language and the
program itself, and then see if I can start hitting some of the forum
requests.
My forum name is "BursegSardaukar"
http://www.slightlymagic.net/forum/memberlist.php?mode=viewprofile&u=23398

**Comment by LevelX2** [comment]
I have sent yo a pm on forum with profile data.

**Pull request merged**